solutions/devicesimulation/single-vm/docker-compose.yml

```diff
@@ -134,10 +134,12 @@ services:
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
-      - PCS_DEVICESIMULATION_WEBSERVICE_URL
+      - PCS_DEVICESIMULATION_WEBSERVICE_URL=http://devicesimulation:9003/v1
```

Comment:
as discussed offline this will cause circular dependency in DS

Reply:
we already have this circular dependency in Master branch. I will remove this in a separate PR.